LICENSE_BIN

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Did you check that this file becomes part of the package when running npm run package?

Reply:
no

Reply:
OK... Can you add an issue for that? I don't think it is automatic but the license should be part of the distributed package.

Reply:
see https://github.com/letmaik/avatar-animator/issues/7